package.json

```diff
@@ -20,45 +20,46 @@
     "@skyux/i18n": "^3.5.0"
   },
   "dependencies": {
-    "prismjs": "1.17.1"
+    "prismjs": "1.19.0"
```

Comment:
Does this change what languages we support on the code block component? https://developer.blackbaud.com/stache/components/code-block

Reply:
Good question.  Looks like it does based on https://github.com/PrismJS/prism/blob/master/CHANGELOG.md and https://prismjs.com/#supported-languages

We have 151 languages listed while their docs have 208.  I'll address that in a PR.